### DISCUSSION TITLE
Monthpicker (pick month instead of date)

### DISCUSSION THREAD

**Discussion by EliasVal** [gpbl/react-day-picker#1986] [Ideas]
The idea is fairly simple, navigation is by the year, and instead of days, the "panel" would display the months.
As for value, it could return the 1st of said month (ie user picker Jan 1999, returned value would be Jan 1st 1999)

**Comment by gpbl** [comment]
Hi @EliasVal, this feature would be out of the scope of DayPicker. Implementing a month picker should be fairly trivial in React, and you'd get a dependency less in your app!

**Comment by dytra** [comment]
please we need this feature

**Comment by shahbazshueb** [comment]
Please we also need this feature
